config/envs/test.py

```diff
@@ -10,16 +10,4 @@ class TestConfig(DefaultConfig):
 
     # Add any test specific config here
 
-    # Database
-    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL").replace(
-        "postgres://", "postgresql://"
-    )
```

Comment:
@SentientHousePlant - I wasnt sure what this was doing if we set the env var correctly. 
Do I need to keep this in?

Reply:
okay thanks @gidsg ill keep it in

Reply:
okay thanks both. So if PAAS does pass it in correctly, its still safe to have this code in - ill leave it in 👍